roles/dnsmasq/handlers/main.yml

```diff
@@ -7,4 +17,18 @@
       networking
       {%- endif %}
     state: restarted
-  when: ansible_os_family != "CoreOS"
+  register: network_runner
+  async: 600
+  poll: 0
+
+- name: Dnsmasq | Wait for network reloaded
+  vars:
+    ansible_ssh_pipelining: yes
+  async_status: jid={{ network_runner.ansible_job_id }}
+  register: network_jobs
+  until: network_jobs.finished
+  retries: 30
+
+- name: Dnsmasq | update resolvconf maybe
+  command: /sbin/resolvconf -u
+  ignore_errors: true
```

Comment:
I don't like ignoring errors, but I understand skipping this if resolvconf doesn't exist. At least add when: ansible_os_family == "Debian"